src/intrprtr.c

```diff
@@ -2074,6 +2074,31 @@ void            IntrLongFloatExpr (
     PushObj(ConvertFloatLiteralEager(string));
 }
 
+/****************************************************************************
+**
+*F   IntrSmallGAPIntExpr()  . . . . . . .  'interpret' a GAP small integer
+**
+**  'IntrSmallGAPIntExpr' is the action to 'interpret' a existing GAP small
+**  integer. This is used for implementing constants.
+*/
+void IntrSmallGAPIntExpr(Obj val)
```

Comment:
I really prefer to refer to "immediate integers", as "small integer" is quite ambiguous (a reader may not realize it has a technical meaning). So perhaps `IntrImmediateIntExpr` and `CodeImmediateIntExpr`?

Reply:
Or `...IntObj`?